static/xref/script.js

```diff
@@ -80,10 +80,10 @@ const options = {
 };
 
 function getFormData() {
-  const term = form.querySelector("input[name='term']").value;
-  const specs = form.querySelector("input[name='cite']").values;
-  const types = form.querySelector("input[name='types']").values;
-  const forContext = form.querySelector("input[name='for']").value;
+  const term = form.term.value;
```

Comment:
Maybe make the aliasing more obvious?

```suggestion
  const {value: term} = form.term;
```